rustls/src/server/server_conn.rs

```diff
@@ -234,7 +241,9 @@ pub struct ServerConfig<C: CryptoProvider> {
     /// How to produce tickets.
     pub ticketer: Arc<dyn ProducesTickets>,
 
-    /// How to choose a server cert and key.
+    /// How to choose a server cert and key. This is usually set by
```

Comment:
Doesn't rustdoc end up looking nicer if you isolate one sentence on the first line?